src/card/index.js

```diff
@@ -11,28 +11,30 @@ const Card = ({
   wrapContent,
   elevation,
   noPadding,
-  header, ...other
+  header,
+  ...other
 }) => {
-  const classes = classnames(theme.card, theme[`elevation-${elevation}`], {
-    [theme.wrapContent]: wrapContent,
-    [theme.noPadding]: noPadding,
-  }, className);
+  const classes = classnames(
+    theme.card,
+    theme[`elevation-${elevation}`],
+    {
+      [theme.wrapContent]: wrapContent,
+      [theme.noPadding]: noPadding,
+    },
+    className,
+  );
   const headerClass = classnames(theme.cardHeader);
 
   let cardHeader;
   if (typeof header === 'string') {
-    cardHeader = (
-      <div className={headerClass}>
-        <span>{header}</span>
-      </div>
-    );
+    cardHeader = <span>{header}</span>;
```

Comment:
Do you think adding a class to this span to style would help ? Or let the custom styling through the prop itself (By sending JSX) ?

Reply:
this was actually wrapped by a div that I've moved to the render method for writing render tests where I've used aria-label prop for verifying that header and its children are properly rendered or not.